lib/array_equals.rb

```diff
@@ -1,5 +1,26 @@
 # Determines if the two input arrays have the same count of elements
 # and the same integer values in the same exact order
 def array_equals(array1, array2)
-  raise NotImplementedError
+  if array1 == nil || array2 == nil
+    if array1 == nil && array2 == nil
```

Comment:
This whole block takes the form of:

`if (condition)
  return true
else
  return false
end`

Simplify to: `return (condition)`

The latter is more readable and easier to type.